pom.xml

```diff
@@ -252,27 +230,15 @@
     <ee8.weld.version>3.1.9.Final</ee8.weld.version>
 
     <ee9.jakarta.activation.api.version>2.0.1</ee9.jakarta.activation.api.version>
-    <ee9.jakarta.annotation.api.version>2.0.0</ee9.jakarta.annotation.api.version>
-    <ee9.jakarta.authentication.api.version>2.0.0</ee9.jakarta.authentication.api.version>
-    <ee9.jakarta.el.api.version>4.0.0</ee9.jakarta.el.api.version>
-    <ee9.jakarta.enterprise.cdi.api.version>3.0.1</ee9.jakarta.enterprise.cdi.api.version>
+    <ee9.jakarta.bom.version>9.1.0</ee9.jakarta.bom.version>
     <ee9.jakarta.inject.api.version>2.0.1.MR</ee9.jakarta.inject.api.version>
```

Comment:
Not done yet.
These need to be removed from the top level pom as well.

Reply:
hehe that's my question here https://github.com/jetty/jetty.project/issues/11976#issuecomment-2854047279
I would prefer too. 
@janbartel ?

Reply:
properties moved